external/storm-kafka/src/jvm/org/apache/storm/kafka/bolt/KafkaBolt.java

```diff
@@ -51,7 +51,9 @@
  * This bolt uses 0.8.2 Kafka Producer API.
  * <p/>
  * It works for sending tuples to older Kafka version (0.8.1).
+ * @deprecated Please use the KafkaBolt in storm-kafka-client
  */
+@Deprecated
```

Comment:
I would rather do that in a follow on JIRA.  Most of the code is the same between the two so getting the rework from here to make there is a lot simpler if I can do a cherry pick